Src/Installer/Dasher.wxs

```diff
@@ -17,12 +17,10 @@
 				<Directory Id="Dasher" Name="Dasher">
 					<Directory Id="INSTALLDIR" Name="Dasher 5.00">
 						<Component Id="MainExecutable" Guid="5a93ba53-2e2d-4f88-8a1f-5bdd8f130394">
-							<File Id="DasherEXE" Name="Dasher.exe" DiskId="1" Source="..\Win32\Target\Release\Dasher.exe" Vital="yes"/>
```

Comment:
We could change it easily because there are examples in .wxs file shows how to do that. My concern here is we need to change the installation steps in that way. If it's okay we can refer to environment variables. 

Also I will try to change them to Release and build with release configuration. If it works we can change it to way it was before.

Reply:
I changed them to Release, as it was at the start. Also we can add env. variables as you said.

Reply:
Now I Changed it to an environment variable.

If "DASHERCONFIG" is "Release" then it works for release, if it's "Debug" then it works for debug. Also as you can see on https://ci.appveyor.com/project/erdemkiraz/dasher I created two jobs, for debug and release. Release took 5 mins and Debug took 10 mins. If one of them is enough, we can decrease it two one job, with deleting one configuration from matrix. 

I think it's better now. Just we have to mention DASHERCONFIG in Installation guide. And currently I'm working on libexpat.dll issue, but there are lots of files interested in $EXPAT. If I can do that, I will add it too.